improver_tests/cli/test_init.py

```diff
@@ -348,5 +348,13 @@ def test_mismatched_close_brackets(self):
             unbracket(['foo', ']', 'bar'])
 
 
+def test_import_cli():
+    """Test if `import improver.cli` pulls in heavy stuff like numpy."""
```

Comment:
Suggest adding another line or two here explaining why we want to avoid large library imports in `improver.cli`.
For example:
> Large imports cause no-operation (eg. --help or exiting due to incorrect arguments) improver CLI usage to be unnecessarily slow.